src/engraving/libmscore/keysig.cpp

```diff
@@ -315,6 +315,19 @@ void KeySig::draw(mu::draw::Painter* painter) const
     painter->setPen(curColor());
     for (const KeySym& ks: _sig.keySymbols()) {
         drawSymbol(ks.sym, painter, PointF(ks.pos.x(), ks.pos.y()));
+        // draw ledgerLines
+        qreal _spatium = spatium();
+        qreal xshift = (ks.sym == SymId::accidentalSharp) ? (_spatium * .15) : (_spatium * .25);
```

Comment:
Maybe make this qreal x = ks.pos.x() - (ks.sym == SymId::accidentalSharp) ? (_spatium * .15) : (_spatium * .25); and use that in the 2 drawSymbol() calls?

Reply:
In fact, they are never called both, every time just one of them, so what is the reason for this change? Pretyfng code?

Reply:
Ah, I missed that

Reply:
Won't harm anyway